src/SourceDialog/SourceDialog.vala

```diff
@@ -256,26 +282,32 @@ public class Maya.View.SourceDialog : Gtk.Grid {
             var cal = (E.SourceCalendar)source.get_extension (E.SOURCE_EXTENSION_CALENDAR);
 
             switch (cal.dup_color ()) {
-                case "#da3d41":
-                    color_button_red.active = true;
+                case "#ed5353":
+                    color_button_strawberry.active = true;
                     break;
-                case "#f37329":
+                case "#ffa154":
                     color_button_orange.active = true;
                     break;
-                case "#e6a92a":
-                    color_button_yellow.active = true;
+                case "#f9c440":
+                    color_button_banana.active = true;
                     break;
-                case "#81c837":
-                    color_button_green.active = true;
+                case "#68b723":
```

Comment:
With the current color palette and an assumed foreground color of #fff, at least @LIME_700 is required to provide WCAG compliance.

Reply:
The previous color value was not part of the official color palette. Therefore I changed it at least so that it now matches the specifications in the ColorButton.css.

If I apply the color values according to the ColorButton.css, I currently achieve no WCAG compliance at all. Is such compliance desirable here?

With the current color palette and an assumed foreground color of #fff, at least @LIME_700 is required to provide WCAG compliance.